src/KafkaW/ProducerTopic.cpp

```diff
@@ -6,12 +6,12 @@ namespace KafkaW {
 
 ProducerTopic::ProducerTopic(std::shared_ptr<Producer> ProducerPtr,
                              std::string TopicName)
-    : KafkaProducer(ProducerPtr), Name(std::move(TopicName)) {
+    : KafkaProducer(ProducerPtr), Name(std::move(TopicName)),
+      ConfigPtr(RdKafka::Conf::create(RdKafka::Conf::CONF_TOPIC)) {
```

Comment:
Would initialising it in the header be better than initialising it here?

e.g. `std::unique_ptr<RdKafka::Conf> ConfigPtr{RdKafka::Conf::create(RdKafka::Conf::CONF_TOPIC)};` or something using make_unique.

Reply:
Sure.